src/openrct2/peep/Peep.cpp

```diff
@@ -359,7 +359,7 @@ void Peep::SwitchToSpecialSprite(uint8_t special_sprite_id)
     SpecialSprite = special_sprite_id;
 
     // If NONE_1 or NONE_2
```

Comment:
```suggestion
    // If Idle or Walking
```
We can either fix the comment or delete it altogether, your call